src/caret_analyze/infra/lttng/ros2_tracing/data_model.py

```diff
@@ -896,13 +896,13 @@ def finalize(self) -> None:
         self.tilde_subscribe_added = self._tilde_subscribe_added.get_finalized('subscription_id')
         del self._tilde_subscribe_added
 
-        self.buffer_to_ipbs = self._buffer_to_ipbs.get_finalized()
+        self.buffer_to_ipbs = self._buffer_to_ipbs.get_finalized('ipb')
```

Comment:
I suggest changing index of buffer_to_ipbs to 'buffer' because 'buffer' is the key of this data.
```suggestion
        self.buffer_to_ipbs = self._buffer_to_ipbs.get_finalized('buffer')
```

Reply:
Fixed